common/windows/cpp/include/registry.h

```diff
@@ -110,6 +110,10 @@
 #define REGSZ_KeyboardHotkeysAreToggle "hotkeys are toggles"
 #define REGSZ_DeadkeyConversionMode    "deadkey conversion mode"                // CU   // I4552
 #define REGSZ_ZapVirtualKeyCode        "zap virtual key code"   // LM, defaults to 0x0E (_VK_PREFIX_DEFAULT)
+/* Non-chiral use of hotkeys instead of left-only hotkeys */
+#define REGSZ_UseRightModifierHotKey "use right modifier for hotkey"
```

Comment:
```suggestion
#define REGSZ_AllowRightModifierHotKey "allow right modifier for hotkey"
```

Definitely a nit but I think a little clearer than 'use'?